lib/IMAP/ImapMessageFetcher.php

```diff
@@ -525,6 +525,10 @@ private function parseHeaders(Horde_Imap_Client_Data_Fetch $fetch): void {
 			$listHeaders = new Horde_ListHeaders();
 			/** @var Horde_ListHeaders_Base[] $headers */
```

Comment:
this should probably go. it my erase the false variant of the union return type of `parse`